spec/raven/configuration_spec.rb

```diff
@@ -130,4 +130,16 @@
     end
   end
 
+  context 'configuration for sanitize fields' do
+    it 'should union default sanitize fields with user-defined sanitize fields' do
+      fields = Raven::Processor::SanitizeData::DEFAULT_FIELDS | %w(test monkeybutt)
```

Comment:
Right, but I'm adding those to the list of default fields and assigning all of that to `sanitize_fields`.